testkit/gradle-testkit-support/src/main/kotlin/com/autonomousapps/kit/render/Scribe.kt

```diff
@@ -85,4 +85,8 @@ public class Scribe @JvmOverloads constructor(
   }
 
   private fun quote(): String = if (dslKind == GradleProject.DslKind.GROOVY) "'" else "\""
+
+  internal fun build(): String {
+    return buffer.toString()
+  }
```

Comment:
what is this for?

Reply:
I used this in the SettingScript in order to build an unindented line and would enable using the quoting type that is currently tied to the DslKind. 

That is:
- Groovy = `'`
- Kotlin = `"`

I'll admit that this is probably a less desired solution to knowing what kind of quote to use based on DslKind 🤦🏼‍♂️

However, if for portability between languages, `"` would be agreed on as the quoting type to use universally, then this is completely unnecessary.